e2ev2/src/circle-run-tests.sh

```diff
@@ -16,4 +16,27 @@ export UI_HOSTNAME=pr-"$PR_SITE_NUM"-dot-all-of-us-workbench-test.appspot.com
 export PUPPETEER_EXECUTABLE_PATH=/usr/bin/google-chrome
 export JEST_SILENT_REPORTER_DOTS=true
 export JEST_SILENT_REPORTER_SHOW_PATHS=true
-yarn test --reporters=jest-silent-reporter
+
+BKT_ROOT=gs://all-of-us-workbench-test.appspot.com/circle-failed-tests
+gsutil cp $BKT_ROOT/\*.$CIRCLE_SHA1.txt failed-tests.txt || true
+gsutil rm $BKT_ROOT/\*.$CIRCLE_SHA1.txt || true
+
+set +e
```

Comment:
This turns off the "fail immediately" bit so we can capture the test result and save the result before reporting.

Reply:
Read the doc at the top of the file on "Unofficial Bash Strict Mode." It's great.